index.html

```diff
@@ -1,158 +1,132 @@
+                    <div class="logo"></div>
+                    <nav>
+                        <ul class="nav__list">                   
+                            <li class="nav__list-item">   
+                                <a class="active__link" href="#">Stays</a>
+                            </li>
+                            <li class="nav__list-item">
+                                <a class="menu__link" href="#">Attractions</a>
+                            </li>   
+                            <li class="nav__list-item">
+                                <a class="menu__link" href="#"></a>
+                            </li>
+                            <li class="nav__list-item">
+                                <a class="menu__link" href="#"></a>
+                            </li>
+                        </ul>
+                    </nav>
+                </div>
+            </div> 
+            
+            <div class="header__content">
+                <h1>Discover stays <span> to live,work or just relax</span></h1>
+                <form action="/">
+                    <div class="header__input--search">
+                        <p>
+                            <label for="search">Your destination or hotel name</label>
+                        </p>
+                        <input type="text" name="destination" id="destination" placeholder="New York">
+                    </div>
+                    <div class="header__input--check">
+                        <p>
+                            <label for="in">Check-in - Check-out</label>
+                        </p>
+                        <input type="text" name="date" id="date" placeholder="Tue 15 Sept &ndash; Sat 19 Sept" onfocus="(this.type='date')" onblur="(this.type='text')">
+                    </div>
+                    <div class="header__input--persons">
+                        <input class="persons" placeholder="2 Adults &ndash; 0 Children &ndash; 1 Room"/>   
+                    </div>
+                    <button class="header__btn">Search</button>
+                </form>
+                
+                <div class="markets">
+                    <div class="android">
+                        <img src="/img/google-play.svg" alt="play market">
+                    </div>
+                    <div class="ios">
+                        <img src="/img/App-store.svg" alt="app store">
+                    </div>
+                </div>
+            </div>    
+        </div>
+
+    </header>
+
+    <section class="offer">
 
-<table width="50%" border="2" cellpadding="5"> 
-    <thead>
-        <tr>
-            <th>Tag</th>
-            <th>Description</th>
-        </tr>    
-    </thead>
+        <div class="container">
+            <h2>What do we offer</h2>
+            <div class="line"></div>
+                <ul class="offer__list">
+                    <li class="offer__item">
+                        <div class="offer__icon-headphone"></div>
```

Comment:
you need put img in this div and add common class for div and add a height to it, which is equal to the height of the maximum image or even a little more